pkg/build/builder/daemonless.go

```diff
@@ -160,7 +160,7 @@ func mergeNodeCredentialsDockerAuth(credsPath string) *docker.AuthConfigurations
 }
 
 func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName string, searchPaths []string, blobCacheDirectory string) error {
-	log.V(0).Infof("Asked to pull fresh copy of %q.", imageName)
+	log.V(2).Infof("Asked to pull a fresh copy of %q.", imageName)
```

Comment:
```suggestion
	log.V(2).Infof("Attempting pull of image %q.", imageName)
```

Reply:
I see this is back to v2.  Did you confirm we're printing v2 by default?  Or that the dptp team is prepared to configure the loglevel for their builds explicitly to get this output if they want it?

Reply:
the latter, at least initially ... in fact, they have to go to V6 to get the info down in the buildah layer that they need

Reply:
in that case while i think this is still worth cleaning up, it seems like there's no need to backport it for them?

Reply:
actually the thought was having a start/end around calling into buildah would help both them, as well as build api team, to more easily delineate "we've dropped down into vendored containers code, we've come back up from vendored containers code"

I'd like to backport this to 4.7, as that is where testplatform will be investigating the current slowness, an soliciting build api and buildah teams for diagnostics

Reply:
but it's not actually adding any net new logging they don't already get today (just changing the wording), so what's the value in backporting it?

they can already see the start/end of the pull w/ the existing logging.

Reply:
that was not clear from the last time we looked at this, at least from the sense that there was not a mixing of build image concerns and buildah concerns in those start/end logs

there was a clear start log, but no end log that I saw, or see now looking at the code

but maybe I am missing it ... where do thing the explicit "end" log is for pulling ImageSource entries?

Reply:
i mean this PR is not adding any net net log statements.  So either:

1) the existing log statements map to begin/end of the pull (even if the wording isn't perfect) and there's no need to backport the wording cleanup to 4.7

or

2) this PR isn't sufficient to address the ask (to add explicit start/end log statements)

what am i missing?

Reply:
Well, upon review, I think it is fair to say the PR title is misleading.  The "start" log already exists.  Though we tweaked the wording with this PR.  So https://github.com/openshift/builder/pull/236/files#diff-9a18a571061b068db5c93ccfb3b354c59d32cb835224bbee929350034fb7fb92R163

now, in the original PR, I originally had a new log right before https://github.com/openshift/builder/pull/236/files#diff-9a18a571061b068db5c93ccfb3b354c59d32cb835224bbee929350034fb7fb92R203 but Adam said that was redundant to https://github.com/openshift/builder/pull/236/files#diff-9a18a571061b068db5c93ccfb3b354c59d32cb835224bbee929350034fb7fb92R163 in that nothing of consequence happened between those 2 lines

And this PR then adds the end log at https://github.com/openshift/builder/pull/236/files#diff-9a18a571061b068db5c93ccfb3b354c59d32cb835224bbee929350034fb7fb92R205

I don't see any other existing logs that demarcate ^^ explicitly ... that more than anything is what would help, so we can distinguish clearly when pulling the ImageSources end and the rest of the build commences.

I'll update the commit and PR title around "adding" exit, "tweaking" start messages.

Does that help?

Reply:
> And this PR then adds the end log at https://github.com/openshift/builder/pull/236/files#diff-9a18a571061b068db5c93ccfb3b354c59d32cb835224bbee929350034fb7fb92R205

this PR isn't adding that line, though.  it's already there(this PR just adds a conditional around it and changes the loglevel).  Or you're saying that line isn't in 4.7 today?  that might be the detail i was missing.  If it's not in 4.7 today then yeah, backporting makes sense.

Reply:
yes sorry that is it ... we are "cleaning" up the logs we originally introduced in 4.8, per you comments on "making them professional" in the 4.7 PR, v2 is the default, etc.

once this merges, I'll manually pick this commit and the prior commit to make sure the final result here in 4.8 lands in 4.7